src/mono/wasm/Wasm.Build.Tests/Blazor/IcuTests.cs

```diff
@@ -0,0 +1,132 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using Xunit;
+using Xunit.Abstractions;
+using Xunit.Sdk;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+#nullable enable
+
+namespace Wasm.Build.Tests.Blazor;
+
+// these tests only check if correct ICU files got copied
+public class IcuTests : BlazorWasmTestBase
+{
+    public IcuTests(ITestOutputHelper output, SharedBuildPerTestClassFixture buildContext)
+        : base(output, buildContext) {}
+
+    [Theory]
+    [InlineData("Debug", false)]
+    [InlineData("Debug", true)]
+    [InlineData("Debug", null)]
+    [InlineData("Release", false)]
+    [InlineData("Release", true)]
+    [InlineData("Release", null)]
+    public async Task HybridWithInvariant(string config, bool? invariant)
+    {
+        string id = $"blz_hybrid_{config}_{GetRandomNameWithoutDots()}";
```

Comment:
Use the recently added `GetRandomId()` which will give you an id usable as an identifier.